docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO driver`, `iRMC driver` and `Redfish driver`.
+
+Check also
+
+- [Ironic UEFI secure-boot](https://docs.openstack.org/ironic/latest/admin/security.html#uefi-secure-boot-mode)
+- [Wikipeda UEFI secure-boot](https://en.wikipedia.org/wiki/UEFI#SECURE-BOOT)
+
+## Advantages/Benefits
+
+We need the automatic secure boot when provisioning a host with high security requirements. Based on checksum and signature, the secure boot protects the host from loading malicious code in the boot process before loading the provisioned operating system.
+
+## Usage
+
+To enable Automatic secure boot check that the hardware is supported then specify the value `UEFISecureBoot` for `bootmode` in the bmh custom resource. Check the example:
+
+```YAML
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: node-1
+spec:
+  online: true
+  bootMACAddress: 00:5c:52:31:3a:9c
+  bootMode: UEFISecureBoot
+  ...
+```
+
+This will enable UEFI in before booting the instance and disabled when deprovisioned. Note that the default value for `bootMode` is `UEFI`.
```

Comment:
```suggestion
This will enable UEFI before booting the instance and disable it when deprovisioned. Note that the default value for `bootMode` is `UEFI`.
```